source/ranges.tex

```diff
@@ -330,6 +330,15 @@
     inline constexpr auto @\libmember{values}{views}@ = elements<1>;
   }
 
+  template<class C, @\libconcept{input_range}@ R, class... Args> requires (!@\libconcept{view}@<C>)
+    constexpr C to(R&& r, Args&&... args);
+  template<template<class...> class C, @\libconcept{input_range}@ R, class... Args>
+    constexpr auto to(R && r, Args&&... args) -> @\seebelow@;
+  template<class C, class... Args> requires (!@\exposid{view}@<C>)
+    constexpr auto to(Args&&... args) -> @\seebelow@;
+  template<template<class...> class C, class... Args>
+    constexpr auto to(Args&&... args) -> @\seebelow@;
```

Comment:
```suggestion
  // \ref{range.utility.conv}, range conversions
  template<class C, @\libconcept{input_range}@ R, class... Args> requires (!@\libconcept{view}@<C>)
    constexpr C to(R&& r, Args&&... args);
  template<template<class...> class C, @\libconcept{input_range}@ R, class... Args>
    constexpr auto to(R && r, Args&&... args) -> @\seebelow@;
  template<class C, class... Args> requires (!@\exposid{view}@<C>)
    constexpr auto to(Args&&... args) -> @\seebelow@;
  template<template<class...> class C, class... Args>
    constexpr auto to(Args&&... args) -> @\seebelow@;
```
In addition, P1206R7's choice to put this inside a bunch of range adaptors seems weird... I think we should move this up to directly after dangling iterator handling, so that both paragraph number and consistency with chapter sequence is maintained.

Reply:
fixed